cronjobs/memberInfo.js

```diff
@@ -0,0 +1,39 @@
+const https = require('https');
+
+function postData(users) {
+  const options = {
+    host: 'api.gaymers.gg',
+    path: '/userdata',
+    method: 'POST',
+    headers: {
+      'x-api-key': process.env.APIGW_DISCOBOT_X_API_KEY
+    }
+  };
+
+  const request = https.request(options);
+
+  request.write(JSON.stringify(users));
+  request.end();
+}
+
+module.exports = {
+  process: (bot) => {
+    console.log('Running memberInfo cron');
```

Comment:
I tried to remove all other `console.log` in the rest of the codebase since they weren't particularly helpful.